getPopsure/dirty-swan
# feat(badge): Add Badge component

### What this PR does
This PR adds the `Badge` component.
This also removes the legacy CSS badge story.

**Preview:**
<img width="1056" alt="Screenshot 2023-07-13 at 15 36 45" src="https://github.com/getPopsure/dirty-swan/assets/4015038/143036f0-32a3-434e-a21a-ebddac626854">

### How to test?
Run storybook and check http://localhost:9009/?path=/docs/jsx-badge--badge-story

### Checklist:

  Or give a reason why this does not apply:
  Or give a reason why this does not apply:

### Browser support

My code works in the following browsers:
